packages/flutter/lib/src/gestures/drag_details.dart

```diff
@@ -71,6 +71,7 @@ class DragStartDetails {
     this.sourceTimeStamp,
     this.globalPosition = Offset.zero,
     Offset? localPosition,
+    this.kind,
```

Comment:
I usually worry about adding parameters like this, but I think it's ok in this case for DragStartDetails since TapDownDetails also has it 👍